server/apps/main/templates/main/deletion_success.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+
+{% load static %}
+
+<head>
+  <!-- Required meta tags -->
+  <meta charset="utf-8">
+  <meta content="width=device-width, initial-scale=1, shrink-to-fit=no" name="viewport">
+
+  <!-- Bootstrap CSS -->
+  <link crossorigin="anonymous" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
+        integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" rel="stylesheet">
+
+  <!--Google Fonts-->
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,400;0,700;1,300&display=swap"
+        rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=PT+Serif:400,700,400italic" rel="stylesheet" type="text/css">
+
+  <!--  Stylesheet-->
+  <link href="{% static '/css/confirmation_page.css' %}" rel="stylesheet">
+  <title>View Stories</title>
+</head>
+<body>
+{% include 'main/partials/navigation.html' %}
+
+<section class="confirmation">
+  <div class="confirmation-container">
+    <div class="confirm_success">
+      <h1 class="big-heading">Success!</h1>
+      <p class="title-text">Experience '{{title}}' does not exist for the current user</p>
```

Comment:
This wording is a bit confusing to me, when I played around with it and encountered the `does not exist for the current user` wording I assumed I ran into an error, rather than success. But I think this is always displayed on deletion of a public experience?

Reply:
This will also be solved by #411